Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -8238,10 +8251,23 @@ def get_whois(domain, normalized=None):
     return parse_raw_whois(raw_data, normalized=normalized, never_query_handles=False,
                            handle_server=server_list[-1])
 
+
 # Drops the mic disable-secrets-detection-end
 
+def get_domain_from_query(query):
+    # checks for largest matching suffix inside tlds dictionary
+    suffix_len = max([len(suffix) for suffix in tlds if query.endswith('.{}'.format(suffix))] or [0])
```

Comment:
Shouldn't this be:
```suggestion
    suffix_len = max([len(suffix) for suffix in tlds if query.endswith('.{}'.format(suffix))] or [])
```

Reply:
No, tried it. max function in python 2.7 doesn't support an empty list. and doesn’t have a 'default' keyword parameter.

Reply:
than i think it should be just 0, as you check the len, not list

Reply:
I tried it also and it failed with `'int' object is not iterable`. if the list is empty it requires another list to compare and run max function on.